sources/packages/backend/apps/api/src/services/student-file/student-file.service.ts

```diff
@@ -44,24 +48,35 @@ export class StudentFileService extends RecordDataModelService<StudentFile> {
     studentId: number,
     auditUserId: number,
   ): Promise<StudentFile> {
+    try {
+      // File upload to the S3 file storage.
+      await this.uploadFileToStorage(createFile);
+    } catch (error: unknown) {
+      throw new CustomNamedError(
+        `Unexpected error while uploading the file ${createFile.fileName}.`,
+        FILE_SAVE_ERROR,
+        error,
```

Comment:
The object info is not meant to be used as a `cause`.
Since the error is already logged we can remove here.